src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -45,6 +45,16 @@ public static Exception GetException(SecurityStatusPal status)
         internal const bool StartMutualAuthAsAnonymous = true;
         internal const bool CanEncryptEmptyMessage = true;
 
+        // used to disable TLS resume on Windows
+        private static readonly Interop.SChannel.SCHANNEL_SESSION_TOKEN s_sessionToken = new Interop.SChannel.SCHANNEL_SESSION_TOKEN
+        {
+            dwTokenType = Interop.SChannel.SCHANNEL_SESSION,
+            dwFlags = Interop.SChannel.SSL_SESSION_DISABLE_RECONNECTS,
+        };
```

Comment:
is this used only to initialize `s_sessionTokenBuffer` below? Maybe we can move it to a helper initialization function to avoid keeping memory.

Reply:
I don't understand the comment.  I made it static instead off the original processing.

Reply:
I was thinking of something like

```csharp
private static readonly byte[] s_sessionTokenBuffer = MemoryMarshal.AsBytes(new ReadOnlySpan<Interop.SChannel.SCHANNEL_SESSION_TOKEN>(new Interop.SChannel.SCHANNEL_SESSION_TOKEN
        {
            dwTokenType = Interop.SChannel.SCHANNEL_SESSION,
            dwFlags = Interop.SChannel.SSL_SESSION_DISABLE_RECONNECTS,
        };)).ToArray();
```

So that we don't keep a separate `Interop.SChannel.SCHANNEL_SESSION_TOKEN` which is not used anymore at runtime.

Reply:
updated.